src/framework/mlt_frame.h

```diff
@@ -150,6 +150,8 @@ extern int mlt_image_format_size( mlt_image_format format, int width, int height
 extern const char * mlt_audio_format_name( mlt_audio_format format );
 extern int mlt_audio_format_size( mlt_audio_format format, int samples, int channels );
 extern void mlt_frame_write_ppm( mlt_frame frame );
+extern int mlt_image_format_planes( mlt_image_format format, int width, int height, void* data, unsigned char *planes[4], int strides[4]);
```

Comment:
I still think this would be more useful as three functions:

```
mlt_image_get_plane_pointer()
mlt_image_get_plane_width()
mlt_image_get_plane_height()
```

But I don't feel strongly about it.

To further my point that this function is ffmpeg specific, MLT does not have any planar formats with three planes. So planes[4] could be planes[3]. But then it would not work with ffmpeg functions.